R/plot-build.r

```diff
@@ -261,6 +266,70 @@ ggplot_gtable.ggplot_built <- function(data) {
   plot_table <- gtable_add_grob(plot_table, caption, name = "caption",
     t = -1, b = -1, l = min(pans$l), r = max(pans$r), clip = "off")
 
+  plot_table <- gtable_add_rows(plot_table, unit(0, 'pt'), pos = 0)
+  plot_table <- gtable_add_cols(plot_table, unit(0, 'pt'), pos = 0)
+  plot_table <- gtable_add_rows(plot_table, unit(0, 'pt'), pos = -1)
+  plot_table <- gtable_add_cols(plot_table, unit(0, 'pt'), pos = -1)
+
+  tag_pos <- theme$plot.tag.position
+  if (length(tag_pos) == 2) tag_pos <- "manual"
+  stopifnot(tag_pos %in% c("topleft", "top", "topright", "left", "right",
```

Comment:
Can you please make sure you use modern tidyverse style everywhere?

Reply:
The error has been improved in e439a97 - are there other issues with styling? (couldn't find any)